### PR TITLE
(SIMP-952) Default to encrypted ISO disks

### PR DESCRIPTION
**Pull request by trevor-vaughan**
SIMP-5817 #close
SIMP-5818 #close

### CODE REVIEW

File: build/distributions/CentOS/6/x86_64/DVD/isolinux/boot.msg [hunk 1]
```diff
@@ -7,7 +7,7 @@
     - This will erase your existing system!
     - You need at least 50GB of disk space for this to succeed.
     - Partitions will be configured to be STIG compliant.
-    - Add the option 0asimp_disk_crypt07 to encrypt your disk.
+    - Type 0asimp-nocrypt07 to disable disk encryption.
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/6/x86_64/DVD/isolinux/boot.msg]: I think this description is confusing.  I would rather see a description that started with "To auto-install SIMP, you can type \<ENTER\>, simp \<ENTER\>, or simp-nocrypt \<ENTER\>" and bullets that explain that \<ENTER\> and simp \<ENTER\> are identical and provide disk encryption, whereas simp-nocrypt \<ENTER\> does not.

File: build/distributions/CentOS/6/x86_64/DVD/isolinux/boot.msg [hunk 2]
```diff
@@ -17,4 +17,4 @@
     - This will erase your existing system!
     - You need at least 50GB of disk space for this to succeed.
     - Partitions will be configured to be STIG compliant.
-    - Add the option 0asimp_disk_crypt07 to encrypt your disk.
+    - Type 0alinux-min-nocrypt07 to disable disk encryption.
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/6/x86_64/DVD/isolinux/boot.msg]: Same issue with description as simp and simp-nocrypt above

File: build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg [hunk 3]
```diff
@@ -14,8 +14,16 @@
 #  > ruby -r 'digest/sha2' -e 'puts "password".crypt("$6$" + rand(36**8).to_s(36))'
 #
 # Optionally:
-# - Adjust the bootloader line for UEFI boot per inline instructions below
-# - Disable FIPS per inline instructions below
+#   - Enable transparent disk encryption by passing 'simp_disk_crypt' in the
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg]: What does 'transparent disk encryption' mean?  Transparent seems to be the opposite of encrypted.

File: build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg [hunk 4]
```diff
@@ -14,8 +14,16 @@
 #  > ruby -r 'digest/sha2' -e 'puts "password".crypt("$6$" + rand(36**8).to_s(36))'
 #
 # Optionally:
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg]: "Optional:"  would be better pairing with "Required:"

File: build/distributions/CentOS/6/x86_64/DVD/isolinux/isolinux.cfg [hunk 5]
```diff
@@ -2,12 +2,18 @@ default simp
 prompt 1
 display boot.msg
 label simp
+  kernel vmlinuz
+  append ks=cdrom:/ks/dvd/auto.cfg initrd=initrd.img audit=1 simp_install=auto simp_disk_crypt
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/6/x86_64/DVD/isolinux/isolinux.cfg]: OCD nit:  Can you please put simp_disk_crypt before simp_install=auto to match the boot option order shown in the UEFI menus?

**Comment by lnemsick-simp** [comment]
This should not be merged until after SIMP 6.3.1, as it requires simp-packer changes to be made.

File: build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg (hunk 3 above)
Comment (sharkbruhaha) [reply build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg]: Transparent as in the user doesn't notice that it is there, and the files are encrypted, not just the store.

File: build/distributions/CentOS/6/x86_64/DVD/isolinux/boot.msg (hunk 1 above)
Comment (trevor-vaughan) [reply build/distributions/CentOS/6/x86_64/DVD/isolinux/boot.msg]: @lnemsick-simp If it gets much longer than this, it doesn't fit on the screen.

File: build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg (hunk 3 above)
Comment (trevor-vaughan) [reply build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg]: Yup, transparent as in "meets the technical requirement if you need it but it isn't great"

File: build/distributions/CentOS/6/x86_64/DVD/isolinux/boot.msg (hunk 1 above)
Comment (trevor-vaughan) [reply build/distributions/CentOS/6/x86_64/DVD/isolinux/boot.msg]: @lnemsick-simp and @sharkbruhaha Open to rewording that doesn't span more than one line. An additional line will make the messages not fit on a standard terminal IIRC.

**Pull request merged**